plans/phase-1/final-cr-definition.md

```diff
@@ -0,0 +1,690 @@
+# Final Multigres Operator CR Definition
+
+```ascii
+[MultiCluster] 🚀 (The root CR)
+      │
+      ├── 🌍 [GlobalTopoServer] (Child CR if managed)
+      │    │
+      │    └── 🏛️ etcd Resources (if managed)
+      │
+      ├── 🤖 MultiAdmin Resources - Deployment, Services, Etc
+      │
+      ├── 💠 [MultiCell] (Child CR)
+      │    │
+      │    ├── 🚪 MultiGate Resources (Deployment, Service, etc.)
+      │    │
+      │    ├── 🧠 MultiOrch Resources (Deployment, etc.)
+      │    │
+      │    └── 📡 [LocalTopoServer] (Child CR if managed and not using global)
+      │         │
+      │         └── 🏛️ etcd Resources (if managed)
+      │
+      └── 🗃️ [MultiTableGroup] (Child CR)
+           │
+           └── 📦 [MultiShard] (Child CR)
+                │
+                └── 🏊 MultiPooler and postgres resources (pods or statefulset)
+```
+
+## MultiGres Cluster
+
+The only editable entry for the end-user. All other child CRs are read-only.
+
+```yaml
+apiVersion: multigres.com/v1alpha1
+kind: MultigresCluster
+metadata:
+  name: example-multigres-cluster
+  namespace: multigres
```

Comment:
Just for clarity, let's use some other namespace in this example. It's surely not about just `multigres` namespace being able to deploy the cluster.